src/include/pagestat_inline.h

```diff
@@ -6,20 +6,23 @@
  * See the file LICENSE for redistribution information.
  */
 
-#define WT_STAT_NONE -1
+#define WT_STAT_NONE UINT64_MAX
```

Comment:
Since now we have WT_STAT_NONE as UINT64_MAX, we can change the definition of stat to use uint64_t.